### PR TITLE
Link fixes

### PR DESCRIPTION
**Pull request by kellyanne-lecuyer**
All links should work for EN and FR.

### CODE REVIEW

File: src/fr/curriculum/index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
 			<p>Les liens qui ne fonctionnent qu'au sein du réseau d'EDSC ou derrière le pare-feu du gouvernement du Canada sont marqués avec <i class="fas fa-external-link-square-alt" aria-hidden="true"> Icône</i><span class="wb-inv">Lien interne</span></p>
 		</div>
 	<!-- DataAjaxFragmentEnd -->
-</div>
+<section>
```
Comment (duboisp) [inline src/fr/curriculum/index.html:11]: I don't think the section element should be used here. The content after seem to belong under the h3 heading "Apprenants cibles"

**Review by duboisp — CHANGES_REQUESTED** [review]

**Comment by duboisp** [comment]
Other that removing the section element, I am fine with the changes in this PR

**Comment by shawnthompson** [comment]
I'm going to merge this and we can fix anything in a new pull request.

**Comment by shawnthompson** [comment]
@kellyanne-lecuyer can you resolve this conflict?

**Review by shawnthompson — APPROVED** [review]
Looks good

File: src/fr/curriculum/index.html (hunk 1 above)
Comment (shawnthompson) [reply src/fr/curriculum/index.html]: Removed all sections

**Pull request merged**